src/components/ImportAddressesButton.jsx

```diff
@@ -117,6 +118,7 @@ function ImportAddressesButton({ addresses = [], client, importCallback }) {
         await importCallback(successes, rescan);
       }
     } catch (e) {
+      console.log("e:", e);
```

Comment:
I'm surprised this isn't an eslint error :thinking:

Reply:
Sorry, yeah this was just for testing. I will remove. Lint gave a warning for this locally rather than an error so that's probably why it passed. Something we might want to change?